rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -131,6 +131,14 @@ Requires: kernel-utils
 Requires: dmidecode
 %endif
 
+# https fails on old distro because they don't support modern certificates (namely rhel3 and sles10)
```

Comment:
don't we need to the the default value to true?

Reply:
it can be anything, the default within the makefile is true since we compare to the litteral "false"